src/v0/destinations/am/utils.js

```diff
@@ -82,6 +82,31 @@ const getEventId = (payload, sourceKey) => {
   return undefined;
 };
 
+/**
+ * generates the unsetObject and returns it 
```

Comment:
is there doc link in amplitude website for this?

Reply:
(https://www.docs.developers.amplitude.com/analytics/apis/http-v2-api/#keys-for-the-event-argument:~:text=exceed%2040%20layers.-,user_properties,-Optional.%20Object.%20A)

Reply:
Let's add reference documentation in source-code itself ?

Reply:
Done